### PR TITLE
Roc version upgrade

### PR DESCRIPTION
**Pull request by Naycon**
Updates the plugin to be compatible with the latest version of roc.

### CODE REVIEW

File: extensions/roc-plugin-typescript/src/roc/index.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import webpack from '../webpack';
+
+export default {
+    description: `
+    # roc-plugin-typescript
```
Comment (dlmr) [inline extensions/roc-plugin-typescript/src/roc/index.js]: The name is automatically added, we so not need to add it manually here.

File: extensions/roc-plugin-typescript/src/roc/index.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    \`\`\`
+
+    The plugin will automatically hook into webpacks build process and add a loader
+    for *.ts and *.tsx files. The compiler will look for a \`tsconfig.json\` file in
+    the root of you roc project, which you can use for configuration.
+    `,
+    actions: [{
+        hook: 'build-webpack',
+        description: 'Adds typescript support to Webpack.',
+        action: webpack,
```
Comment (dlmr) [inline extensions/roc-plugin-typescript/src/roc/index.js]: We can gain some slight performance by using `lazyFunctionRequire` here.

**Comment by dlmr** [comment]
Looks good overall! :+1:

File: extensions/roc-plugin-typescript/src/roc/index.js (hunk 2 above)
Comment (dlmr) [reply extensions/roc-plugin-typescript/src/roc/index.js]: https://github.com/rocjs/roc/blob/master/docs/RocObject.md#tip-for-optimal-performance

**Comment by dlmr** [comment]
If you want to use the linting rules that `rid` provides without needing a `.eslintrc` in the project you can use the `rid lint` command over `rid lint:alias`

File: extensions/roc-plugin-typescript/src/roc/index.js (hunk 1 above)
Comment (Naycon) [reply extensions/roc-plugin-typescript/src/roc/index.js]: Fixed, thanks!

File: extensions/roc-plugin-typescript/src/roc/index.js (hunk 2 above)
Comment (Naycon) [reply extensions/roc-plugin-typescript/src/roc/index.js]: Great! Fixed in latest commits.

**Pull request merged**